finetuner/tuner/base.py

```diff
@@ -186,6 +185,20 @@ def fit(
             a Keras model.
         """
 
+        try:
+            self._fit(
+                train_data,
+                eval_data,
+                epochs,
+                batch_size,
+                num_items_per_class,
+                preprocess_fn,
+                collate_fn,
+                num_workers,
+            )
+        except BaseException as e:
```

Comment:
why not simply Exception?

Reply:
I think `BaseException` is the base class of `Exception`, and some exception directly subclass `BaseException`